contrib/build-static.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash -xeu
+
+REPO_MUSL_COMPAT="https://github.com/juju/musl-compat"
+REPO_LIBTIRPC="https://salsa.debian.org/debian/libtirpc"
+REPO_LIBNSL="https://github.com/thkukuk/libnsl"
+REPO_LIBUV="https://github.com/libuv/libuv"
+REPO_LIBLZ4="https://github.com/lz4/lz4"
+REPO_SQLITE="https://github.com/sqlite/sqlite"
+REPO_DQLITE="https://github.com/cole-miller/dqlite"
+
+TAG_MUSL_COMPAT="main"
```

Comment:
Could you please include a test run against the musl tag used in our k8s distributions: https://github.com/canonical/k8s-dqlite/blob/master/hack/env.sh#L13? We might not always be on the same musl version as the juju team.

Reply:
Juju will be happy to move to the later one, now we know the root cause. We intended to move that originally but ended up locating the segfault.

Reply:
Sounds good.